### DISCUSSION TITLE
[QUESTION]: How to create a queue??

### DISCUSSION THREAD

**Discussion by wpz850398226** [Netflix/conductor#2784] [Q&A]
When I was publish a message in queue to mark a wait task as completed，use /api/queue/update/{workflowId}/{taskRefName}/{status}
I got response such as "There is no queue for handling COMPLETED status",I don't know how to create a queue in conductor,please

**Comment by aravindanr** [comment]
Conductor automatically creates queues for `COMPLETED` and `FAILED` statuses. Do you have any of the following properties set to `true`?

```
conductor.event-queues.sqs.enabled
conductor.event-queues.amqp.enabled
conductor.event-queues.nats.enabled
```

**Comment by wpz850398226** [comment]
![image](https://user-images.githubusercontent.com/41596989/150894634-621aec09-590f-437b-a649-83d9d6f3ae2b.png)
thanks!Should I add **"value": true** here in additional-spring-configuration-metadata.json?

**Comment by aravindanr** [comment]
The `additional-spring-configuration-metadata.json` is a file that describes configuration properties. Conductor deployments typically have a properties file listing the customizations and features enabled. The value for `conductor.event-queues.sqs.enabled` can be set to `true` there.
